SPECS-EXTENDED/suitesparse/suitesparse.spec

```diff
@@ -0,0 +1,752 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
```

Comment:
Since this is a brand new specs, could you lint it even though it's in `SPECS-EXTENDED`?